### ISSUE TITLE
Question: What is the scalability, reliability, availability & cost like?

### ISSUE THREAD

**Issue by mangoplane** [decentralized-identity/ion#329]
Hi,

So I'm curious about stats of the ION node network, which are:

1. Cost to run your own ION Node
2. How many nodes are there, and is it likely at this stage for the P2P network to simply turn off?
3. How long are ION DID's accessible? I understand they're stored in IPFS, which means they're around for as long as they're pinned.

My understanding is ION is just a kind of naming service for IPFS stored DIDs, where I suppose only the person with the credentials to verify their DID can change its contents. This that understanding correct?

I want a long lived way to store public values related to a users credentials in a service I am experiment with, and testing locally. I may never launch this service, it's just a tinkering hobby project at the moment.

Regards.

**Comment by rayie** [comment]
Daniel Buchner speaks about the cost in this video: https://youtu.be/3UbUzA0lW9w?feature=shared 
However, it's from 2020, and I haven't been able to find out the latest on wether ION is still active status.

**Comment by downTheFallLine** [comment]
from what i gather, microsoft has removed ion support from entra.   this is a MAJOR setback for ion world, imo.

**Comment by mentioneddd** [comment]
> from what i gather, microsoft has removed ion support from entra. this is a MAJOR setback for ion world, imo.

yep built an entire project on ion took several months to figure everything out to work - everything goes in the trashbin for me.
